src/main/webapp/app/overview/course-overview.scss

```diff
@@ -162,3 +162,17 @@ Course Info Bar
 canvas#complete-chart {
     z-index: 99;
 }
+
+.dev-button {
+    text-align: left;
+    color: white;
+    background-color: #3e8acc;
+    &:hover {
+        background-color: #0f6ab4;
+    }
+    &:active {
+        box-shadow: 0 2px 0 $artemis-dark;
+    }
+    border: none;
+    border-radius: 1px;
```

Comment:
Maybe increase the border radius a little. 1px is barely noticeable and looks almost square. Also if the buttons are placed without space between them, there should only be a border radius towards the two outer sides and not inbetween buttons (maybe you can use Bootstrap's [button group](https://getbootstrap.com/docs/4.4/components/button-group/) since it handles that out of the box).

Generally it would probably be preferable to use the regular buttons (they have a nicer focus state as well for example). I think they even offer a setting to make them smaller.

Reply:
I will leave the buttons for now like the are currently implemented, because only the developers will see these buttons